CONTRIBUTING.rst

```diff
@@ -18,216 +18,16 @@ Workflow is pretty straightforward:
 
```

Comment:
@asvetlov instead of keeping 2 `contributing.rst` files in repo, it is possible to have just one in `docs/` folder: I've noticed that github picks it up as well now.

Reply:
https://help.github.com/articles/setting-guidelines-for-repository-contributors/

Reply:
I'd like to have sphinx markup in docs version and keep a simple file on top-level for sake of visibility for newcomers.

Reply:
But anyway good to know. Looks like `.rst` is not supported, only Markdown and text.

Reply:
I think I saw RST being supported in some of my projects (JFYI)